backend/src/controllers/sessionsController.js

```diff
@@ -1,2 +1,9 @@
-//Create a controller for managing the sessions that handles the /api/sessions requests.
+import { createToken } from '../services/sessionService';
 
+export const sessionsController = {
+  post( req, res) {
```

Comment:
@LLaszlo89 pay attention to styling
```suggestion
  post(req, res) {
```